http/jaxrs/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,22 @@
+####
```

Comment:
I recall we dropped all the docker files in the OpenShift test suite, so I guess we should not add them. wdty @rsvoboda ?